src/index.ts

```diff
@@ -0,0 +1,9 @@
+export function greet(
+  inputName: string
+): { message: string } | { error: string } {
+  const namePattern = /^[a-zA-Z\s'-]+$/;
```

Comment:
Really cool! I'm personally terrible at regex, so fun to see that you are rocking them during the precourse :-D

I asked Chat GPT if there's any names that wouldn't match, and here's the result: https://chatgpt.com/share/06a181dc-9e4f-4eec-a311-558097666c3c.

It's great to validate, but just be careful! There's probably someone in the world with just spaces as a name.

Reply:
A lot of spaces! :) 
Am also terrible at regex but I use this website [](https://regex101.com/)

Reply:
I enjoyed seeing the regex too. :)  Interresting response from Chat GPT.

@Odin-Taif now you have to exclude José too!